nixos/modules/services/web-apps/lotide.nix

```diff
@@ -0,0 +1,104 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lotide;
+in
+{
+  options.services.lotide = {
+    enable = mkEnableOption "the lotide service";
+
+    package = mkOption {
+      default = pkgs.lotide;
+      defaultText = "pkgs.lotide";
+      type = types.package;
+      description = "lotide package to use.";
+    };
+
+    databaseUrl = mkOption {
+      default = null;
+      type = types.string;
+      description = "Credentials for database connection.";
+      example = "postgresql://user:passwort@somehost:5432/databasename";
+    };
+
+    hostUrlActivityPub = mkOption {
+      default = null;
+      type = types.string;
+      description = "If using the recommended proxy setup, set this to your root address";
```

Comment:
Please consistently use a trailing period for all descriptions, except for those with `mkEnableOption`.